Makefile

```diff
@@ -35,7 +35,9 @@ install: release
 	/bin/chmod -x $(DESTDIR)/usr/lib/flexbridge/*.md*
 	/usr/bin/install lib/common/setup-user.sh $(DESTDIR)/usr/lib/flexbridge
 	/usr/bin/install lib/common/run-app $(DESTDIR)/usr/lib/flexbridge
-	/usr/bin/install -m644 lib/ReleaseMono/Mercurial-$(CPUARCH).zip $(DESTDIR)/usr/lib/flexbridge
+	# Copy mercurial for both architectures since flexbridge is an any architecture package.
```

Comment:
That may be. FB was *just* changed to be an architecture-independent package earlier this year, IIRC. But we could go to the work to change it back.